src/components/20-molecules/accordion/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.2.0
+
+- Add the `ariaLevel` attribute defines the hierarchical level of an element within a structure.
+- Add more HTML semantic and remove all unnecessary DOM Manipulations
```

Comment:
spelling: **semantics** (noun, not adjective), **manipulation** (lowercase, singular)